scripts/plot_operator_breakdown.py

```diff
@@ -15,48 +15,48 @@
 import matplotlib.colors as mplcolors
 from matplotlib.colors import LinearSegmentedColormap
 
-if(not len(sys.argv) in [1,2]):
+if not len(sys.argv) in [1, 2]:
```

Comment:
I saw changes from `not x in y` to `x not in y` above. Were those manual? If not, why didn't the linter complain here?

Reply:
According to that [rule](https://www.flake8rules.com/rules/E713.html), all occurrences should have been changed. Apparently, it does not recognize that case, maybe because of the call to `len`. I will replace it manually.